semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/Utils.java

```diff
@@ -0,0 +1,26 @@
+package org.hypertrace.semantic.convention.utils;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Splitter;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class Utils {
```

Comment:
`Utils` is a super generic name. Can we have a more descriptive name for the Utils class? Just like we have `HttpSemanticConventionUtils` and so on

Reply:
@rish691 and I discussed this and thought to name this. This file just has one method which is used by two files so named this

Reply:
How about SanitizerUtils?

Reply:
Ig this would not work if more methods are added in future. Not sure tho.